packages/__docs__/src/App/index.tsx

```diff
@@ -123,36 +124,12 @@ class App extends Component<AppProps, AppState> {
       versionsData: null,
       iconsData: null
     }
-
-    this._content = null
-    this._menuTrigger = null
-    this._mediaQueryListener = null
-
-    this.fetchVersionData()
-    fetch('icons-data.json')
-      .then((response) => response.json())
-      .then((iconsData) => {
-        this.setState({ iconsData: iconsData })
-      })
-    fetch('markdown-and-sources-data.json')
-      .then((response) => response.json())
-      .then((docsData) => {
-        this.setState(
-          {
-            docsData,
-            themeKey: Object.keys(docsData.themes)[0]
-          },
-          this.scrollToElement
-        )
-      })
-      .catch((error) => {
-        // eslint-disable-next-line no-console
-        console.error('Unable to load docs data :(\n' + error)
-      })
   }
```

Comment:
These had to be moved out from the constructor since these fetch calls are side effects and we have to be able to cancel them.